commands/publish/index.js

```diff
@@ -144,10 +144,58 @@ class PublishCommand extends Command {
         )
       : [this.packagesToPublish];
 
+    const isBreakingChange = (currentVersion, nextVersion) => {
+      const releaseType = semver.diff(currentVersion, nextVersion);
+      switch (releaseType) {
+        case "major":
+          return true;
+        case "minor":
+          return semver.lt(currentVersion, "1.0.0");
+        case "patch":
+          return semver.lt(currentVersion, "0.1.0");
+        case "premajor":
+        case "preminor":
+        case "prepatch":
+        case "prerelease":
+          return false;
+        default: {
+          const err = { error: "not expected release type" };
+          throw err;
```

Comment:
This should throw a new Error instance, but I'm not really sure how we get to this case anyway, because `semver.diff()` already throws when its arguments are invalid versions.